docs/gen-ai/README.md

```diff
@@ -7,9 +7,24 @@ linkTitle: Generative AI
 **Status**: [Development][DocumentStatus]
 
 > [!Warning]
-> The semantic conventions for GenAI and LLM are currently in development.
-> We encourage instrumentation libraries and telemetry consumers developers to
-> use the conventions in limited non-critical workloads and share the feedback
+>
+> Existing GenAI instrumentations that are using
+> [v1.34.0 of this document](https://github.com/open-telemetry/semantic-conventions/blob/v1.34.0/docs/gen-ai/README.md)
+> (or prior):
+>
+> * SHOULD NOT change the version of the GenAI conventions that they emit by default.
+>   Conventions include, but are not limited to, attributes, metric, span and event names,
+>   span kind and unit of measure.
+> * SHOULD introduce an environment variable `OTEL_SEMCONV_EXPERIMENTAL_OPT_IN`
+>   as a comma-separated list of category-specific values. The list of values
+>   includes:
+>   * `gen-ai-latest` - emit the latest (supported by the instrumentation) GenAI
+>     conventions and dot not emit the old one (v1.34.0 or prior).
+>   * The default behavior (in the absence of one of `gen-ai-latest`) is to continue
+>     emitting whatever version of the GenAI conventions the instrumentation
+>     was emitting (1.34.0 or prior).
+> * SHOULD update the baseline version to a stable one in the next major version
+>   after GenAI conventions are declared stable.
```

Comment:
I found this a bit confusing without listing the `genai` (stable) value above

Reply:
resolving, this was also due to me missing the word EXPERIMENTAL in the env var

Reply:
I'd prefer not to add anything about stable versions until we have stability in sight. So I removed this point and added 

> This transition plan will be updated to include stable version before the
> GenAI conventions are marked as stable.